lib/cisco_node_utils/cmd_ref/interface.yaml

```diff
@@ -52,6 +56,19 @@ encapsulation_dot1q:
   config_set_append: "%s encapsulation dot1q %s"
   default_value: ""
 
+encapsulation_profile_vni:
+  _exclude: [/N3/, /N5K/, /N6K/, /N9K/]
+  multiple:
+  config_get_token_append: [
+    '/^service instance (%s) vni/',
```

Comment:
I considered it but I can't without changing every other property as well. We should definitely do this, but not here.